tests/components/backup/test_websocket.py

```diff
@@ -2261,7 +2281,6 @@ async def test_config_schedule_logic(
             1,
             {
                 "test.test-agent": [call("backup-1")],
-                "test.test-agent2": [call("backup-1")],
```

Comment:
This changes since we now always count backups per agent instead of counting backups regardless of agents. Agents may now keep different backups but will honor the set number of backups to keep.